src/styles/main.ts

```diff
@@ -1,87 +1,124 @@
-import { StyleSheet, ViewStyle, TextStyle, Dimensions, ImageStyle, Platform } from "react-native";
+import { StyleSheet, ViewStyle, TextStyle, Dimensions, ImageStyle, Platform } from 'react-native';
 
 const DEVICE_WIDTH = Dimensions.get('window').width;
 const DEVICE_HEIGHT = Dimensions.get('window').height;
 
+const paragraph = {
+  paddingTop: 10,
+  paddingBottom: 10,
+  paddingLeft: 20,
+  paddingRight: 20,
+} as TextStyle;
+
+const errorMessage = Object.assign({}, paragraph, {
```

Comment:
It doesn't change anything, but for future reference, you could have used the spread operator to archive the same and it looks a little bit nicer:
```
const errorMessage = {
  ...paragraph,
  color: '#FF4833'
};
```